src/javascript/app/Stores/Modules/Trading/trade-store.js

```diff
@@ -419,6 +422,8 @@ export default class TradeStore extends BaseStore {
             this.proposal_requests = requests;
             this.proposal_info     = {};
             this.purchase_info     = {};
+            this.root_store.modules.contract.is_digit_contract =
+                isDigitContract(Object.keys(this.proposal_requests)[0]);
```

Comment:
is this necessary?

Reply:
yes, when exiting the contract mode, we need to set this based on the current proposal's contract_type